lib/scutil/lockfile.c

```diff
@@ -54,6 +59,9 @@ static char *udir = NULL;
  */
 static long uwaiting;
 
+#ifdef _WIN32
+#define pid_t int
```

Comment:
Microsoft marked POSIX function names as deprecated like getpid, they recommend to use '_' version of them, could we use _getpid() and define it here?

Reply:
I don't think so, because this is a primarily linux target. I think that if ms ever removes those functions, we can just define them again.